bin/node-template/runtime/src/lib.rs

```diff
@@ -43,6 +40,13 @@ pub use sp_runtime::{Perbill, Permill};
 /// Import the template pallet.
 pub use pallet_template;
 
+// Use own values for `BlockExecutionWeight`, `ExtrinsicBaseWeight`, `RocksDbWeight`
+// and `ParityDbWeight` instead the defaults from [`frame_support::weights`].
+pub mod weights;
+
+// Re-export the non-default weights.
+pub use weights::*;
```

Comment:
Id rather we dont have this re-export for clarity where things are coming from

Reply:
Maybe we could rename the weights.rs to template_weights.rs instead?